src/client/components/Fields.tsx

```diff
@@ -1,4 +1,8 @@
-import { UserDiversityOptions, UserYearOptions, UserGenderOptions, UserPronounOptions, UserShirtSizeOptions } from '@Shared/UserEnums';
+import { UserDiversityOptions,
+  UserYearOptions,
+  UserGenderOptions,
+  UserPronounOptions,
+  UserShirtSizeOptions } from '@Shared/UserEnums';
```

Comment:
Can these be aligned with line 1? Does ESLint fix these kind of formatting issues?

Since this is a refactoring PR I'm gonna be ocd about these kind of stuff 😛